domains/loganpenguino.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "LoganPenguino",
+    "email": "badah-bunny@gmail.com"
+  },
+  "records": {
+    "CNAME": "https://loganpenguino.github.io/log/"
```

Comment:
Invalid CNAME. CNAME cannot contain protocols, file paths, URL parameters, or anything combination of those. See docs for more information: https://docs.is-a.dev/ Reopen PR when this is fixed.